### PR TITLE
Add Buildpacks community organization RFC

### PR DESCRIPTION
**Pull request by samj1912**
Signed-off-by: Sambhav Kothari <skothari44@bloomberg.net>

### CODE REVIEW

File: text/0000-buildpacks-community.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+# Prior Art
+[prior-art]: #prior-art
+
+- [CNCF Sandbox](https://www.cncf.io/sandbox-projects/)
+- [Paketo Community](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0008-paketo-community.md)
+- [Argoproj Labs](https://github.com/argoproj-labs)
+- [Crossplane Contrib](https://github.com/crossplane-contrib)
+
+# Unresolved Questions
```
Comment (jkutner) [inline text/0000-buildpacks-community.md:131]: should we move the [pack-orb](https://github.com/buildpacks/pack-orb) here?

**Review by jkutner — APPROVED** [review]

File: text/0000-buildpacks-community.md (hunk 1 above)
Comment (samj1912) [reply text/0000-buildpacks-community.md:131]: I think so. I also think we might want to move the tekton plugin here as well?

Comment (samj1912) [reply text/0000-buildpacks-community.md:131]: @AidanDelaney I might propose moving scafall here as well to allow you to be unblocked.

**Review by AidanDelaney — APPROVED** [review]
This looks great.

File: text/0000-buildpacks-community.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+## Project Admission
+
+A project can be admitted to the Buildpacks community organization by creating a Github issue in the Buildpacks community repository. The issue should contain the following information:
+
+- Name of the project
+- Evidence for the above criteria
+- A list of maintainers for the project
+
+The above information will be structured into an appropriate issue template. The Buildpacks Leadership will review the issue and if the project meets the above criteria, the project will be added to the Buildpacks community organization. The Buildpacks Leadership will assign a steward team to the project and the team lead of the steward team will be responsible for ensuring that the project meets the above criteria.
```
Comment (natalieparellano) [inline text/0000-buildpacks-community.md:88]:
```suggestion
The above information will be structured into an appropriate issue template. The Buildpacks Leadership will review the issue and if the project meets the above criteria, the project will be added to the Buildpacks community organization. The Buildpacks Leadership will assign a team to the project and the team lead of the team steward the project - i.e., will be responsible for ensuring that the project meets the above criteria.
```
IMO this makes it clearer (assuming I understood what you meant)

**Review by natalieparellano — APPROVED** [review]

**Comment by samj1912** [comment]
Moving to voting period ending on Feb 3rd 2023.

**Review by hone — APPROVED** [review]
I think when we pen this in the community repo, we'll need to expand/clarify the archive/removal process a bit more on expectations. I also want to make sure this org doesn't turn into a graveyard of projects.

**Comment by samj1912** [comment]
All binding votes have been cast - merging.

**Pull request merged**